index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Hello World</title>
+    <link href="estilos/estilo.css" rel="stylesheet">
+  </head>
+  <body>
+    <h1 id="initial-text">Oii</h1>
+    <img src="imagens/git-search.PNG" alt="Minha imagem de teste" class="center">
```

Comment:
Neste caspo temos um ótimo SEO aplicado. Ainda sim podemos aprimorar para utilizar o alt e o atributo title.

https://yoast.com/image-seo-alt-tag-and-title-tag-optimization/